exchanges/poloniex/poloniex.go

```diff
@@ -690,7 +690,7 @@ func (e *Exchange) GetMarginAccountSummary(ctx context.Context) (Margin, error)
 }
 
 // PlaceMarginOrder places a margin order
-func (e *Exchange) PlaceMarginOrder(ctx context.Context, currency string, rate, amount, lendingRate float64, buy bool) (OrderResponse, error) {
+func (e *Exchange) PlaceMarginOrder(ctx context.Context, currencyPair string, rate, amount, lendingRate float64, buy bool) (OrderResponse, error) {
```

Comment:
suggestion: could be pair, or symbol across this file for standardisation but up to you.